src/hotspot/share/opto/output.cpp

```diff
@@ -35,8 +35,8 @@
 #include "memory/allocation.hpp"
 #include "opto/ad.hpp"
 #include "opto/block.hpp"
-#include "opto/c2compiler.hpp"
 #include "opto/c2_MacroAssembler.hpp"
+#include "opto/c2compiler.hpp"
```

Comment:
Hmm.
```
#include "opto/c2_MacroAssembler.hpp"
#include "opto/c2compiler.hpp"
```
Here _ comes before lower-case letters. From other places I see that _ comes after upper-case letters. I realize that this is caused by the ASCII value, but it is a bit unfortunate (IMHO). OTOH, maybe this will be consistent (the way I would like it, at least) if the sorting was done on lower-cased strings.

Reply:
I think it's simplest to follow ASCII sorting but don't have a strong opinion if others would prefer for strings to be lower-cased before sorting. The important thing is that the same order is used everywhere.

Reply:
I register my vote for using lower-casing the strings before sorting.

Note that lower-casing before sorting retain the current prevailing sort-order of sorting `_` before the letters. This is also what you get if you use java's `String.CASE_INSENSITIVE_ORDER` (but not if you use `sort -f`, which upper-cases the strings).